instrumentation/rack/lib/opentelemetry/instrumentation/rack/middlewares/event_handler.rb

```diff
@@ -52,9 +52,12 @@ class EventHandler
           # @param [Rack::Response] This is nil in practice
           # @return [void]
           def on_start(request, _)
-            return if untraced_request?(request.env)
+            parent_context = if untraced_request?(request.env)
```

Comment:
Is this relying on start_span to look for the untraced context as opposed to seething the current context to untraced without an active span?

Reply:
Yeah it's seeing that it's parent context is untraced so it will create a non recording span, and we have a lot of `recording?` checks in this instrumentation as is so it seemed like a simpler implementation than what I had in this commit https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/968/commits/a8548efd6ba065ce27da5bcf9d21504efa40b3c1

Reply:
Understood. Thanks again for fixing this!